user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/OntologyFilterConnector.java

```diff
@@ -42,5 +45,12 @@ public void initComboBoxWithOntologyDatasource(
                 .collect(Collectors.toList())).stream().map(entity -> OntologyClassDTO.from(entity)),
         entity -> entity
     );
+    box.setRenderer(createRenderer());
   }
+
+  private Renderer<OntologyClassDTO> createRenderer() {
```

Comment:
```suggestion
  private Renderer<OntologyClassDTO> ontologyComponentRenderer() {
```
NIT: you can be more specific here 😄